Tasks/Common/securefiles-common/securefiles-common.ts

```diff
@@ -12,7 +13,19 @@ export class SecureFileHelpers {
         let serverCreds: string = tl.getEndpointAuthorizationParameter('SYSTEMVSSCONNECTION', 'ACCESSTOKEN', false);
         let authHandler = vsts.getPersonalAccessTokenHandler(serverCreds);
 
-        this.serverConnection = new vsts.WebApi(serverUrl, authHandler);
+        let proxy = tl.getHttpProxyConfiguration();
+
+        let option: IRequestOptions = {
+            proxy: {
```

Comment:
It looks like the type returned by `getHttpProxyConfiguration` is duck-type compatible with the type of this property.  So you should be able to just do `proxy: proxy`